Xamarin.Forms.Platform.iOS/Renderers/ListViewRenderer.cs

```diff
@@ -639,23 +642,23 @@ public UnevenListViewDataSource(ListViewDataSource source) : base(source)
 			{
 			}
 
-			internal void CacheDefinedRowHeights()
-			{
-				Task.Run(() =>
```

Comment:
Hopefully this work will also fix the bug that necessitated the introduction of this logic. This logic _asynchronously_ pre-computes the heights of cells. The idea is that the computation is done faster than the user scrolls so the heights are available by the time the user scrolls to the item.